tests/deploy/test.js

```diff
@@ -15,24 +15,21 @@ async function main() {
   await testSimple(net, "minimal");
 
   await testArguments(net, "regular", "ValueWithConstructor");
-  await testArguments(net, "upgradeable", "ValueWithInitializer");
-
-  await assert.rejects(
-    testArguments(net, "upgradeable", "ValueWithConstructor")
-  );
+  await testArguments(net, "upgradeable", "ValueWithConstructor");
```

Comment:
Why not test constructor arguments for `minimal` deploy?

Reply:
Pretty sure that would be covered by tests @frangio is doing for `deploy` as a  separate PR.